.github/workflows/upstream-build.yml

```diff
@@ -57,3 +57,4 @@ jobs:
             EXTRA_TWISTER_FLAGS="--short-build-path -O/tmp/twister-out"
           fi
           west twister -T ../zephyr/samples -s sample.basic.helloworld -p siwx917_rb4338a -v --inline-logs $EXTRA_TWISTER_FLAGS
+          west twister -T ../zephyr/samples -s sample.net.wifi -p siwx917_rb4338a -v --inline-logs -K $EXTRA_TWISTER_FLAGS
```

Comment:
The `-T ../zephyr/samples` is likely unnecessary, but that can be fixed separately as the line above also has the same issue.